public/splashscreen.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="">
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+  <link rel="icon" href="<%= BASE_URL %>favicon.ico" />
+  <title>Flux</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@junto-foundation/junto-elements@0.0.9/dist/main.css" />
+  <script src="https://cdn.jsdelivr.net/npm/@junto-foundation/junto-elements@0.0.9"></script>
```

Comment:
Do we want to use same styling with Flux app?

Reply:
probably not, but would work in the mean time we have proper design.